doc/rbd/index.rst

```diff
@@ -4,18 +4,19 @@
 
 .. index:: Ceph Block Device; introduction
 
-A block is a sequence of bytes (often 512).
-Block-based storage interfaces are a mature and common way to store data on
-media including HDDs, SSDs, CDs, floppy disks, and even tape.
-The ubiquity of block device interfaces is a perfect fit for interacting
-with mass data storage including Ceph.
-
-Ceph block devices are thin-provisioned, resizable, and store data striped over
-multiple OSDs.  Ceph block devices leverage
+RBD (the Ceph block device) is a service that provides thin-provisioned,
+resizable block devices that offer snapshotting and cloning as features. 
+
+A block is a sequence of bytes (often 512). Block-based storage interfaces are
```

Comment:
I might remove `A block is a sequence of bytes (often 512). `, I'm not sure that it adds value here.

Reply:
I agree. I've removed it.

In the main, I don't like this paragraph or the sentences in it. The first sentence points up the maturity and prevalence of the method of block-based storage and then lists physical media upon which block storage has been implemented. Even if I agreed with the inclusion in this introduction to RBD of this idea regarding the pervasiveness of block devices, I question the value in 2022 of emphasizing the maturity of the block-based storage model. I think it puts the text in danger of pointing out what is taken by most people nowadays as a matter of course and therefore unremarkable. And I object outright to the use of the adjective "common" where "prevalent" is meant, but that's just because "common" has a potentially negative connotation if it is accidentally read as referring to class and not read as referring to frequency of use or implementation. It's a very nit-picky semantic distinction, but if we can avoid this kind of ambiguity ,why shouldn't we? I might recast this sentence charitably by writing "Block-based storage interfaces are used to store data on HDDs, SSDs, CDs, floppy disks, and tape. The block storage model, having existed for decades, is mature and reliable. Block storage is perhaps the most prevalent form of storage in the early twenty-first century.".

The sentence that follows is a real howler: 

>The ubiquity of block device interfaces is a perfect fit for interacting with mass data storage including Ceph.

I think that this means to say nothing more grand than that Ceph can work with block device interfaces. But it includes this three-pipe problem of a phrasal verb, "is a perfect fit for interacting with", which seems to me the kind of thing that no one could have actually intended to write. Admittedly, I can think of situations in which the phrase "perfect fit" is appropriate: *This shirt is a perfect fit!* or *This couch is a perfect fit, right between the piano and the door!*. But I remain adamant that the sentence as it is written, in which the ubiquity of block device interfaces is identified as a perfect fit for interacting with mass data storage including Ceph, cannot be what was meant. That's just not how the semantics of the word "fit" work. This sentence sounds a bit to me like a Ceph salesman slapping the side-panel of a brand-new, shiny, red Ceph fresh from the factory and on the showroom floor and saying "the ubiquity of block device interfaces makes this baby a perfect fit for interacting with mass data storage including this!". I think that this kind of expression should cause all native English speakers to pause and then to ask "The *ubiquity* does that?".

Reply:
You aren't wrong.